specification/consumption/resource-manager/Microsoft.Consumption/stable/2017-11-30/examples/UsageDetailsExpand.json

```diff
@@ -4,7 +4,7 @@
     "scope": "subscriptions/subid/providers/Microsoft.Billing/billingPeriods/201702",
     "$expand": "meterDetails,additionalProperties",
     "$filter": "usageEnd le 2017-02-14T00:00:00Z",
-    "$top": "1"
+    "$top": 1
```

Comment:
Dont change this version of the swagger/examples. You need to change it in 2018-01-31 version

Reply:
Thanks for catch. This change is reverted, and new one on latest version is added